### PR TITLE
Calendar at bottom of home page

### PR DESCRIPTION
**Pull request by Kstrella**
Got the calendar to show but still need api integration

### CODE REVIEW

**Review by ricardocarva — APPROVED** [review]

File: AgileMindsUI/AgileMindsUI/AgileMindsUI.csproj [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'https' " />
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'http' " />
   <ItemGroup>
+    <PackageReference Include="Heron.MudCalendar" Version="2.0.3" />
```
Comment (roc-ricardo) [inline AgileMindsUI/AgileMindsUI/AgileMindsUI.csproj:17]: @Kstrella, can you do me a favor and remove this line from the commit as the package is used in the AgileMindsUI.Client project?

File: AgileMindsUI/AgileMindsUI/Components/_Imports.razor [hunk 2]
```diff
@@ -10,4 +10,4 @@
 @using MudBlazor.Services
 @using AgileMindsUI
 @using AgileMindsUI.Client
-@using AgileMindsUI.Components
+@using AgileMindsUI.Components
```
Comment (roc-ricardo) [inline AgileMindsUI/AgileMindsUI/Components/_Imports.razor:13]: @Kstrella, can you do me a favor and remove this line from the commit as the package is used in the AgileMindsUI.Client project?

File: AgileMindsUI/AgileMindsUI/Components/_Imports.razor [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿@using System.Net.Http
+﻿﻿@using System.Net.Http
```
Comment (roc-ricardo) [inline AgileMindsUI/AgileMindsUI/Components/_Imports.razor:1]: Is this needed for this Project as the changes of this commit were to the AgileMindsUI.Client?

**Review by roc-ricardo — CHANGES_REQUESTED** [review]

**Comment by roc-ricardo** [comment]
@Kstrella great stuff, can you please commit the changes and push it again please? Once merged, I will move them to the Projects.razor page that we're currently using in `main`.

**Pull request merged**